store/migration/prod/1.10/0005##data_source.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE data_source RENAME COLUMN host_override TO host;
+ALTER TABLE data_source RENAME COLUMN port_override TO port;
+ALTER TABLE data_source ADD COLUMN database TEXT NOT NULL DEFAULT '';
+
+UPDATE data_source
+SET
+	host=(SELECT host FROM instance WHERE id = instance_id),
+	port=(SELECT port FROM instance WHERE id = instance_id),
+	database=(SELECT database FROM instance WHERE id = instance_id)
+WHERE type = 'ADMIN';
```

Comment:
do we also need to backfill the database field for for READ ONLY data source?

Reply:
The database field wasn't available to RO database, so it's not needed.

Reply:
Currently, data source uses the instance database name, which is natural because of the parent-child relationship.

If we move the database name field to the data source,  do we want the readonly data source to use the admin data source connecting with "ALL database names"?